src/components/meeting.jsx

```diff
@@ -85,12 +100,21 @@ export default class Meeting extends Component {
 							<div className="list-group-item">
 								<h5>{meeting.location}</h5>
 								<p className="my-0 mt-1">{meeting.formatted_address}</p>
-								<p className="my-0 mt-1">Other meetings at this address:</p>
+							</div>
+							<div className="list-group-item">
+								<h5>Other Meetings Here:</h5>
 								<ol className="my-0 mt-1">
 									{other_meetings.map(other_meeting => {
+										if(loop_today != other_meeting.day) {
+											loop_today = other_meeting.day;
+											return (
+												<strong>{strings[settings.days[other_meeting.day]]}</strong>
+											)
```

Comment:
This is not good, because it means that the first meeting each day will be skipped. when you do a `return ()` it won't process the other `return ()` later in the function. Also we need a unique `key` in the parent element that's returned in a `map()` so that's also an issue. I think I will rewrite this to iterate over `settings.days` and use a filter to check if there are any meetings.

Reply:
Gah, I'm still figuring out scope within ReactJS, thanks for catching this one... and for teaching me along the way. I'll have to repay the favor with Python some day. :)